Tools/autotest/common.py

```diff
@@ -196,7 +196,10 @@ def fetch_parameters(self):
     def reboot_sitl(self):
         """Reboot SITL instance and wait it to reconnect."""
         self.mavproxy.send("reboot\n")
-        self.mavproxy.expect("tilt alignment complete")
```

Comment:
this is still send on SUB

Reply:
Is it sent when INS_GYR_CAL is set to 0 (our default)?

Reply:
It is not sending here, I'll try to find out why

Reply:
@jaxxzer, I don't know. But that is not the default param that we get on SITL.
Currently it is send. I agree that the caching of the text that we currently have isn't great...
I will be quite busy on the next two weeks but I will make a good pass on Sub simulation and autotest after... it is quite behind the others...

Reply:
It seems to be a parameters issue, I'm taking a further look

Reply:
From what I've gathered testing this, the "tilt alignment complete" message should be sent regardless of parameters, or at least the ones mentioned so far right now.

This message not appearing during certain boots seems to have something to do with running with `--speedup 10`. It appears consistently with speedup 1 (default).